assets/javascripts/back_to_top.js

```diff
@@ -0,0 +1,17 @@
+function backToTop() {
+    $(document).ready(function() {
+        $(window).scroll(function() {
+            if ($(this).scrollTop() > 50) {
+                $('#back-to-top').fadeIn();
+            } else {
+                $('#back-to-top').fadeOut();
+            }
+        });
+        $('#back-to-top').click(function() {
+            $('#back-to-top').tooltip('hide');
+            $('body, html').animate({ scrollTop: 0 }, 800);
```

Comment:
800 feels quite slow (but that's of course a matter of preference).

Reply:
if that is the same value than on cavil then it's good :)

Reply:
Yes, i did not change any of the values from Cavil/QAMDashboard.

Reply:
@okurz "cavil" means making an unnecessary objection so for the sake of irony I say your statement doesn't count.